pkg/poller/prometheus.go

```diff
@@ -24,6 +25,12 @@ func (u *UnifiPoller) RunPrometheus() error {
 		ReportErrors: true, // XXX: Does this need to be configurable?
 	}))
 
+	version.Branch = Branch
+	version.BuildDate = BuildDate
+	version.Revision = Revision
+	version.Version = Version
+	prometheus.MustRegister(version.NewCollector("unifi_poller"))
```

Comment:
I really wish you could name it with a dash here. this bugs me haha

Reply:
Dashes are not allowed in the metric name. :)